jenkins-scripts/docker/lib/generic-abi-base.bash

```diff
@@ -68,13 +68,7 @@ echo '# BEGIN SECTION: compile and install branch: ${DEST_BRANCH}'
 cp -a $WORKSPACE/${ABI_JOB_SOFTWARE_NAME} /tmp/${ABI_JOB_SOFTWARE_NAME}
 chown -R root:root /tmp/${ABI_JOB_SOFTWARE_NAME}
 cd /tmp/${ABI_JOB_SOFTWARE_NAME}
-if ${GITHUB}; then
-  git fetch origin
-  git checkout origin/${DEST_BRANCH}
-else
-  hg pull
-  hg up ${DEST_BRANCH}
-fi
+git checkout origin/${DEST_BRANCH}
```

Comment:
the `git fetch origin` line is removed; is that intentional?

Reply:
it was not!  47a364 . Good catch